.gitignore

```diff
@@ -1,5 +1,6 @@
 .*
 !.gitignore
+!.travis.yml
```

Comment:
why adding this?

Reply:
Because travis.yml should not be ignored :) but it is by default because of .*